### ISSUE TITLE
Team charter outline

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#21]
Description:
Create a team charter outline that will be the foundation for a team charter exercise on 7/29/20.

[Outline of items we want to charter](https://teams.microsoft.com/_#/docx/viewer/teams/https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA~2FShared%20Documents~2FGeneral~2FTeam%20Meetings%20+%20Presentations~2FTeam%20Charter%20and%20Norms.docx?threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&baseUrl=https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA&fileId=A7F48EDC-8E8E-45F7-AAE8-57AD4E9C3C57&ctx=files&viewerAction=edit)

[Possible template for activity  ](https://app.mural.co/t/officeoffamilyassistance2744/m/gsa6/1593121051008/9fe08cfdf876940d181c6a19684eaf4e65381eb3)

AC: 

- [x] Modified team charter template is created and shared w/ PO and team for feedback

Tasks:

- [x] @SelenaJV @shubhi-raft Outline charter elements
- [x] Conduct chartering activity with team
- [x] Incorporate into charter
- [x] Develop review process
- [x] Team members ratify

**Comment by SelenaJV** [comment]
@shubhi-raft I just added the team charter we used for the 18F + OFA team to the MS Team doc linked above. I adapted it a bit to our new bigger team and added some other stuff we may want to incorporate. Totally open to adding/editing any of this, but thought it could be a helpful starting place for our convo tomorrow.

**Comment by shubhi-raft** [comment]
@SelenaJV thanks! sharing the template we use for creating team charter's based on reverse brainstorming. Here is the [mural link](https://app.mural.co/t/raft2792/m/raft2792/1595715915232/1f691ae0191c083e91d9c08b42bc4310be25d403)

**Comment by SelenaJV** [comment]
@shubhi-raft I was tightening up language around values and lost some of what you wrote because I was expecting track changes to work differently. Sorry! 

I managed to get some of your language back and incorporated but please revise if I missed stuff or lost the spirit of something.

Also, please add the bullets you had in mind for the ZenHub norms?

**Comment by SelenaJV** [comment]
Hi team! @lfrohlich @cmajel @lauraGgit @RafterGit @adamcaron @reitermb @matthewjdoty @dk-ui @carltonsmith 

@shubhi-raft and I cobbled together a [team charter](https://teams.microsoft.com/_#/docx/viewer/aggregatefiles/https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA~2FShared%2520Documents~2FGeneral~2FTeam%2520Meetings%2520+%2520Presentations~2FTeam%2520Charter%2520and%2520Norms.docx?baseUrl=https:~2F~2Fhhsgov.sharepoint.com~2Fsites~2FTANFDataPortalOFA&fileId=a7f48edc-8e8e-45f7-aae8-57ad4e9c3c57&ctx=aggregate&viewerAction=view) from our forming activities way back in July and discussions about review process in the past couple weeks. Please give it a read in the next day or two. 

- Check that your info is correct in the Team Members section
- Add comments or questions to anything that doesn't make sense to you
- Add any out of office days you have on your horizon
- Did we miss anything?
- Hold off on "ratifying" until we incorporate any major suggestions 

Thanks all. Hoping this will be helpful.

**Comment by lfrohlich** [comment]
Once #162 is complete (changes from retro incorporated), team members can ratify and this can be closed

**Comment by SelenaJV** [comment]
Reminder: I'm out this Friday and Monday. What do you need from me so this issue can be closed in this sprint?

**Comment by shubhi-raft** [comment]
@SelenaJV this is ready for review. I accepted your comments and add some more comments where text was highlighted or there were question marks.

**Comment by SelenaJV** [comment]
I just read through charter and accepted a bunch of changes. It's looking good!

It looks like you have have a placeholder for the design to dev handoff. I moved it to the end of the issue flow because I think one only encounters this when the design issue is done? Whatever you decide to put there, I think the rest of the team can sign off on the process

Let's push for ratification of whatever we have EOD Monday. If we want to make further changes to charter, it can be a new issue. Let's close this one in Sprint 2! We deserve credit for this work!

**Comment by lfrohlich** [comment]
Team members have ratified.
